spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -30,13 +30,14 @@
   end
 
   def assert_table_counts
-    ContainerGroup.count.should        == 2
-    ContainerNode.count.should         == 1
-    Container.count.should             == 3
-    ContainerService.count.should      == 6
-    ContainerPortConfig.count.should   == 2
-    ContainerDefinition.count.should   == 3
-    ContainerReplicator.count.should   == 2
+    ContainerGroup.count.should      == 2
+    ContainerNode.count.should       == 1
+    Container.count.should           == 3
+    ContainerService.count.should    == 6
+    ContainerPortConfig.count.should == 2
+    ContainerEnvVar.count.should     == 5
+    ContainerDefinition.count.should == 3
```

Comment:
@moolitayer if you touch these ones above (unrelated to the patch), please fix the rubocop warnings (by removing the extra spaces).